public/controllers/register-agent/components/steps-three/group-input.tsx

```diff
@@ -0,0 +1,114 @@
+import React, { Fragment, useState } from 'react';
+import {
+  EuiComboBox,
+  EuiCallOut,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiPopover,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+import { webDocumentationLink } from '../../../../../common/services/web_documentation';
+import { PLUGIN_VERSION_SHORT } from '../../../../../common/constants';
+
+const popoverAgentGroup = (
+  <span>
+    Learn about{' '}
+    <a
```

Comment:
**nitpick**: I think we should use a `EuiLink` component instead of the `a` tag.

Reply:
Resolved in commit [4a0761b](https://github.com/wazuh/wazuh-kibana-app/pull/5554/commits/4a0761b9b9dacd4072886ed4d7c81e3926a368b6)